simularium/simularium-website
# Feature/orthographic UI

Problem
=======
This closes [#279](https://github.com/simularium/simularium-website/issues/279) / [#320](https://github.com/simularium/simularium-website/issues/320) and [#419](https://github.com/simularium/simularium-website/issues/419).

Orthographic UI (#419) was to be implemented in tandem with button styling. In hindsight: review would be easier if this were two PRs, one for styling, one for implementing the camera. Luckily it's not a huge merge.

Solution
========

Camera:
Add UI buttons and props to call setCameraType. (see note about dev build below)

Styling:
Updated colors, size, spacing, and borders of camera and playback controls. Did some cleanup adding .container wrappers on CSS rules and renaming a few things.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
New viewer version released with ortho code!

npm run start
toggle orthographic/perspective modes in the camera controls
check out the styling of camera and playback controls

Screenshots (optional):
----------------
<img width="969" alt="ortho" src="https://github.com/simularium/simularium-website/assets/24981838/db05dc54-6798-4afc-8363-d6808b13a348">
<img width="954" alt="persp" src="https://github.com/simularium/simularium-website/assets/24981838/7b288b01-e73d-4afc-99f7-1aba3507e9f2">
-------
